src/reporters/json.ts

```diff
@@ -379,7 +379,37 @@ export default class JSONReporter extends BaseReporter {
 
     this.runner.on(
       'step:end',
-      ({ journey, step, start, end, error, url, status, metrics }) => {
+      ({
+        journey,
+        step,
+        start,
+        end,
+        error,
+        url,
+        status,
+        pagemetrics,
+        traces,
+        metrics,
+        filmstrips,
+      }) => {
+        this.writeMetrics(journey, step, 'relative_trace', traces);
+        this.writeMetrics(journey, step, 'experience', metrics);
+        if (filmstrips) {
+          // Write each filmstrip separately so that we don't get documents that are too large
+          filmstrips.forEach((strip, index) => {
```

Comment:
Does this affect timing? I thought we saved it for the end on purpose so as not to affect journey timing as much

Reply:
We left it to the end to make sure to run the tracer only once per journey, but we are already running trace for each step anyway so there is no harm in also capturing filmstrips.